clients/vault/src/oracle/agent.rs

```diff
@@ -123,24 +123,12 @@ pub async fn start_oracle_agent(
 			}
 		}
 
+		tracing::info!("start_oracle_agent(): shutting down overlay connection");
 		// shutdown the overlay connection
 		overlay_conn.stop();
 	});
 
-	tokio::spawn(on_shutdown(shutdown_sender.clone(), async move {
```

Comment:
This sometimes work; and sometimes it doesn't. The logs don't show up. 
I decided to remove this and let the `VaultService` perform the `disconnect_signal_sender` .